libs/agno/agno/tools/dalle.py

```diff
@@ -58,17 +59,9 @@ def __init__(
         # - Add support for saving images
         # - Add support for editing images
 
-    def create_image(self, agent: Union[Agent, Team], prompt: str) -> str:
-        """Use this function to generate an image for a prompt.
-
-        Args:
-            prompt (str): A text description of the desired image.
-
-        Returns:
-            str: str: A message indicating if the image has been generated successfully or an error message.
-        """
+    def create_image(self, agent: Union[Agent, Team], prompt: str) -> FunctionCallResult:
         if not self.api_key:
-            return "Please set the OPENAI_API_KEY"
+            return FunctionCallResult(content="Please set the OPENAI_API_KEY")
```

Comment:
We don't have to _always_ use the FunctionCallResult, strings and other types should still be allowed.